src/components/MoneyRequestHeader.js

```diff
@@ -0,0 +1,143 @@
+import Avatar from './Avatar';
+import styles from '../styles/styles';
+import themeColors from '../styles/themes/default';
+import CONST from '../CONST';
+import withWindowDimensions from './withWindowDimensions';
+import compose from '../libs/compose';
+import Navigation from '../libs/Navigation/Navigation';
+import ROUTES from '../ROUTES';
+import Icon from './Icon';
+
+const propTypes = {
+    /** The report currently being looked at */
+    report: iouReportPropTypes.isRequired,
+
+    /** The policies which the user has access to and which the report could be tied to */
+    policies: PropTypes.shape({
+        /** Name of the policy */
+        name: PropTypes.string,
+    }).isRequired,
+
+    /** Personal details so we can get the ones for the report participants */
+    personalDetails: PropTypes.objectOf(participantPropTypes).isRequired,
+
+    /** Whether we're viewing a report with a single transaction in it */
+    isSingleTransactionView: PropTypes.bool,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    isSingleTransactionView: false,
+};
+
+const MoneyRequestHeader = (props) => {
+    const formattedAmount = props.numberFormat(props.report.total / 100, {
+        style: 'currency',
+        currency: props.report.currency,
+    });
+    const isSettled = /* ReportUtils.isSettled(props.report.reportID); */ false;
+    const isExpenseReport = ReportUtils.isExpenseReport(props.report);
+    const payeeName = isExpenseReport
+        ? ReportUtils.getPolicyName(props.report, props.policies)
+        : ReportUtils.getDisplayNameForParticipant(props.report.managerEmail);
+    const payeeAvatar = isExpenseReport
+        ? ReportUtils.getWorkspaceAvatar(props.report)
+        : ReportUtils.getAvatar(lodashGet(props.personalDetails, [props.report.managerEmail, 'avatar']), props.personalDetails);
+    return (
+        <View style={[
+            {backgroundColor: themeColors.highlightBG},
+            styles.pl0,
+        ]}
+        >
+            <HeaderWithCloseButton
+                shouldShowAvatarWithDisplay
+                shouldShowThreeDotsButton={!isSettled}
+                threeDotsMenuItems={[{
+                    icon: Expensicons.Trashcan,
+                    text: props.translate('common.delete'),
+                    onSelected: () => {},
+                }]}
+                threeDotsAnchorPosition={styles.threeDotsPopoverOffsetNoCloseButton}
+                report={props.report}
+                policies={props.policies}
+                personalDetails={props.personalDetails}
+                containerStyles={[styles.pt5, styles.pb3]}
+                shouldShowCloseButton={false}
+                shouldShowBackButton={props.isSmallScreenWidth}
+                onBackButtonPress={() => Navigation.navigate(ROUTES.HOME)}
+            />
+            <View style={[styles.ph5, styles.pb5]}>
+                <Text style={[styles.textLabelSupporting, styles.lh16]}>To</Text>
```

Comment:
We should translate this text

```suggestion
                <Text style={[styles.textLabelSupporting, styles.lh16]}>{props.translate('common.to')}</Text>
```